### PR TITLE
tutorial: Using Cockpit test VMs with your own test framework

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-custom-test-framework.html

### CODE REVIEW

**Comment by martinpitt** [comment]
I think this is the last item I want to salvage from [cockpit-examples](https://github.com/cockpit-project/cockpit-examples) before we mothball it. The pinger test is now in cockpit proper; packaging, most of Makefile, and the cockpit-API tests are are in starter-kit; and the "check-shell" bit is not that useful and also kind of covered in this blog post.

File: _posts/2018-03-28-puppeteer-tests.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+category: tutorial
+tags: cockpit starter-kit tests puppeteer
+slug: cockpit-custom-test-framework
+comments: true
+---
+
+The [Cockpit Starter Kit](https://github.com/cockpit-project/starter-kit/) provides the scaffolding for your own Cockpit
+extensions: a simple page (in React), build system (webpack, babel, eslint, etc.), and an integration test using
+Cockpit's own Python test API (on top of the
+[Chromium DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/).  See the recent
```
Comment (larskarlitski) [inline _posts/2018-03-28-puppeteer-tests.md]: The parentheses starting at "on top of" is not closed after the link here.

Comment (martinpitt) [reply _posts/2018-03-28-puppeteer-tests.md]: Curious, I already had this fixed in my local git, but didn't commit it. Thanks!

Comment (larskarlitski) [reply _posts/2018-03-28-puppeteer-tests.md]: Thanks!

**Pull request merged**